app/controllers/courses_controller.rb

```diff
@@ -152,6 +152,22 @@ def save_unlocks
     redirect_to manage_unlocks_organization_course_path(@organization, @course), alert: e.to_s
   end
 
+  def student_emails
+    authorize! :teach, @organization
+    add_course_breadcrumb
+    add_breadcrumb('Students', students_organization_course_path(@organization, @course))
+    @students = []
+    @course.submissions.each do |s|
+      @students << s.user unless @students.include? s.user
+    end
+
+    respond_to do |format|
+      format.html
+      format.text { render text: @students.map { |s| "#{s.email}" }.join("\n") }
+      format.csv { render text: "Username,Email\n" + @students.map { |s| "#{s.username},#{s.email}"}.join("\n") }
+    end
+  end
+
```

Comment:
Couldn't we easily do this with participants list -- have it courses/:id/participants, with all participants for this course